src/pe.cc

```diff
@@ -0,0 +1,315 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
I was unsure how to handle attribution here, is this good as-is, should this be my name or something else?